vespalib/src/vespa/vespalib/util/array.hpp

```diff
@@ -60,7 +60,9 @@ Array<T>::Array(const Array & rhs)
     : _array(rhs._array.create(rhs.size() * sizeof(T))),
       _sz(rhs.size())
 {
-    construct(array(0), rhs.array(0), _sz, std::is_trivially_copyable<T>());
+    if (_sz > 0) [[likely]] {
```

Comment:
Standard ? Replacement for __builtin_expect..... ?

Reply:
Yep, as of C++20 https://en.cppreference.com/w/cpp/language/attributes/likely